src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,33 @@
 import React from 'react';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
+interface Props {
+  user: User;
+  clear: () => void;
+}
+
+export const CurrentUser: React.FC<Props> = ({ user, clear }) => (
   <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+    <h2 className="CurrentUser__title">
+      <span>
+        Selected user:
+        {user?.id}
```

Comment:
```suggestion
        {user.id}
```
no need ?, because you render CurrenUser only if user exist